README.md

```diff
@@ -184,9 +184,10 @@ Portals come with accessibility defaults right out of the box. Their default ARI
 
 Portals also compute a default label from their embedded contents (by either using the title of the embedded page or concatenating all the visible text in the portal's viewport if the page doesn't have a title). This label can be overridden by authors using the `aria-label` attribute.
 
+The current defaults should ensure that a portal can be accessed by screen readers and have a description without any work from authors. Authors should however implement a click event handler that activates the portal, to ensure they can be activated with screen readers. Portals that are hidden by their embedder until activation time should be marked with `aria-hidden="true"` by authors, so that they override the default behaviour and are invisible to screen readers as well.
```

Comment:
> to ensure they can be activated with screen readers

Well, to ensure they can be activated by anybody, right? Might be worth linking to #174 here.

> Portals that are hidden by their embedder until activation time should be marked with `aria-hidden="true"` by authors

I'd generalize this to say that "should be hidden by authors to avoid interaction (using e.g. the `hidden=""` attribute, or `display: none`)" and then maybe mention that this will also hide them from the accessibility tree. In general, we want accessibility to be more automatic, and not require special `aria-*` attributes to get good behavior. That helps authors more easily create good experiences for all users. I think portals has a good story here, so we shouldn't imply that you need special `aria-hidden=""` management for them.

Reply:
> > to ensure they can be activated with screen readers
> 
> Well, to ensure they can be activated by anybody, right? Might be worth linking to #174 here.
>
Technically yes, but I intended this more for use cases where activation normally happens through some user gesture that isn't easy to reproduce with a screen reader (or isn't intuitive to do when using one). In particular I'm thinking about scroll/swipe triggered activations.

This has got me thinking more about #174 though, and I think that it's a great default from an accessibility point of view.

> > Portals that are hidden by their embedder until activation time should be marked with `aria-hidden="true"` by authors
> 
> I'd generalize this to say that "should be hidden by authors to avoid interaction (using e.g. the `hidden=""` attribute, or `display: none`)" and then maybe mention that this will also hide them from the accessibility tree. In general, we want accessibility to be more automatic, and not require special `aria-*` attributes to get good behavior. That helps authors more easily create good experiences for all users. I think portals has a good story here, so we shouldn't imply that you need special `aria-hidden=""` management for them.

That makes sense, updated.